recipes/pymochi/run_test.py

```diff
@@ -0,0 +1,2 @@
+# check simply importing works
+import pymochi
```

Comment:
This script seems to be redundant, I don't see it being used anywhere in this recipe.